hack/boilerplate/boilerplate.py

```diff
@@ -130,8 +130,8 @@ def file_extension(filename):
     return os.path.splitext(filename)[1].split(".")[-1].lower()
 
 skipped_dirs = ['Godeps', 'third_party', '_gopath', '_output', '.git', 'cluster/env.sh',
-                "vendor", "test/e2e/generated/bindata.go", "hack/boilerplate/test",
-                "pkg/generated/bindata.go"]
+                'vendor', 'test/e2e/generated/bindata.go', 'hack/boilerplate/test',
+                'pkg/generated/bindata.go', 'build/nested-cluster-base']
```

Comment:
:\

Reply:
Agreed. But afaict I can't put the shell scripts in another dir, and I can't slap our boilerplate on someone else's OSS.

Reply:
is this still needed now that you've put the dind stuff in `third_party`?

Reply:
Nope. I've also added an OWNERS file to third_party/moby, so we won't need to bother the top-level approvers in the future.

Reply:
Do we really care about configuring most of these arguments? Why not statically set them?